### PR TITLE
Added attendance list actions

### PR DESCRIPTION
**Pull request by ghulamhafeez**
### Implementation

*  Create attendance action for attendance list 

- For time being I added mock login user data to test my conditions since we don’t have the actual data 

### UI
image
image

### CODE REVIEW

**Review by Mubashar1009 — APPROVED** [review]

**Review by muhammad-bilal-ali-khokhar — COMMENTED** [review]
@ghulamhafeez I'll appricate you If you attach action's screen shoot

**Review by muhammad-bilal-ali-khokhar — COMMENTED** [review]
merge or rebase with main i have update folder structure

File: src/app/attendance/components/AttendanceActionRender.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      onClick: handleRejectRequest,
+      icon: <XCricleIcon size={16} className="text-red-500" />,
+      className: "text-red-500",
+    },
+  ];
+
+  const actionMenu = React.useMemo(() => {
+    switch (loginUser.role) {
+      case MemberRole.Member:
+        return attendanceData.status === "pending" ? [...baseActions] : [];
```
Comment (muhammad-bilal-ali-khokhar) [inline src/app/attendance/components/AttendanceActionRender.tsx:76]: @ghulamhafeez we have enum for `pending`

File: src/app/attendance/components/AttendanceActionRender.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import ActionsMenu from "@/common/ActionMenu/ActionsMenu";
+import { Trash2, Edit, XCircle, CheckSquare } from "lucide-react";
```
[the remark this answers is not in the document]
Comment (ghulamhafeez) [reply src/app/attendance/components/AttendanceActionRender.tsx]: Fixed

File: src/app/attendance/components/AttendanceActionRender.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      title: "Edit Info",
+      id: 2,
+      onClick: handleEditInfo,
+      icon: <Edit size={16} />,
+    },
+    {
+      title: "Delete Member",
+      id: 3,
+      onClick: handleDeleteMember,
+      icon: <Trash2 size={16} />,
```
[the remark this answers is not in the document]
Comment (ghulamhafeez) [reply src/app/attendance/components/AttendanceActionRender.tsx]: Fixed

File: src/app/attendance/components/AttendanceActionRender.tsx (hunk 1 above)
Comment (ghulamhafeez) [reply src/app/attendance/components/AttendanceActionRender.tsx]: Fixed

**Review by muhammad-bilal-ali-khokhar — APPROVED** [review]

**Pull request merged**